src/Jira/Api.php

```diff
@@ -173,14 +173,14 @@ protected function clearLocalCaches()
 	/**
 	 * Get fields definitions.
 	 *
-	 * @return array
+	 * @return array|false
```

Comment:
Valid points, lets investigate that as part of #135.

Here I just made consistent for what we already have, and was not meant to complete unit test coverage for the methods involved, guess that is work for another time :)